modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/PortalConfig.java

```diff
@@ -4,6 +4,8 @@
 {
     String asset_cacheControl() default "public, max-age=31536000, immutable";
 
+    boolean asset_useDefaultContextPathMount() default false;
```

Comment:
context-path mount is not default. It is legacy vhost mount that we don't want to support.

`asset_legacyContextPath() default false`